packages/shared/lib/core/wallet/stores/main-account-id.store.ts

```diff
@@ -0,0 +1,13 @@
+import { get, writable } from 'svelte/store'
+import { updateActiveWalletPersistedData } from '../../profile'
+import { selectedWalletId } from './selected-wallet-id.store'
+
+export const mainAccountId = writable<string | null>(null)
+
+export function updateMainAccountId(payload: string, walletId = get(selectedWalletId)): void {
```

Comment:
we shouldnt be setting this store explicitly, but rather just updating the persisted data and that propagates to our store